roles/manifest/tasks/main.yml

```
@@ -15,4 +15,4 @@
     validate_certs: "{{ foreman_validate_certs | default(omit) }}"
     organization: "{{ foreman_organization }}"
     manifest_path: "{{ foreman_manifest_path }}"
-    state: present
+    state: refreshed
```

Please don't change defaults of existing roles!

This is why the tests fail.

```suggestion
    state: "{{ foreman_manifest_state | default('present') }}"
```

And then you tell users to set it if they want it refreshed.

But honestly, this really doesn't belong into this PR.
If you're uploading a manifest, why do you need a refresh? It should be fresh already.

---

https://github.com/theforeman/foreman-ansible-modules/pull/1286#discussion_r727861792 :smirk_cat: